src/Web/Telegram/API/Bot/API.hs

```diff
@@ -68,182 +73,179 @@ newtype Token = Token Text
 telegramBaseUrl :: BaseUrl
 telegramBaseUrl = BaseUrl Https "api.telegram.org" 443 ""
 
--- | Type for token
-type TelegramToken = Capture ":token" Token
-
 -- | Telegram Bot API
 type TelegramBotAPI =
```

Comment:
This is a good idea. I'm not expert in Servant. Does this allow client code not to have token passed on every line?